.craft.yml

```diff
@@ -7,7 +7,6 @@ changelogPolicy: auto
 targets:
   - name: crates
     noDevDeps: true
-  - name: gh-pages
```

Comment:
Is that because the artifact is missing or are we removing GH pages entirely?

Reply:
That stuff was previously built in a travis job. Its basically duplicated with docs.rs: https://getsentry.github.io/sentry-rust/sentry/

So I think we should remove it in favor of that.

Reply:
Right. I believe we introduced this slightly before docs.rs was a thing but kept it alive. I'm fine with turning off completely 👍